pyanaconda/modules/subscription/initialization.py

```diff
@@ -33,12 +35,26 @@
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
+RHSM_SYSTEMD_UNIT_NAME = "rhsm.service"
+
+
+def check_initial_conditions():
+    """Can the Subscription service run?"""
+
+    # Exclude the dir and image installations.
+    if not conf.target.is_hardware:
+        log.debug("subscription: Unsupported type of the installation target. Quit.")
+        sys.exit(1)
+
+    # Exclude environments without the rhsm service.
+    if not util.is_service_installed(RHSM_SYSTEMD_UNIT_NAME, root="/"):
+        log.debug("subscription: The rhsm service is not available. Quit.")
```

Comment:
Suggestion: ""subscription: The rquired rhsm systemd service is not available, so the Subscription module will not be started.""

Reply:
Fixed.

Reply:
Btw. the amount of the word `subscription` on the logged line starts to be ridiculous. From `/tmp/syslog`:

`
12:18:13,890 WARNING org.fedoraproject.Anaconda.Modules.Subscription: DEBUG: anaconda.modules.subscription.initialization: subscription: The required rhsm systemd service is not available. The Subscription module won't be started.`